src/components/BookContainer.js

```diff
@@ -1,56 +1,58 @@
-import React, { useEffect, useState } from 'react';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import { deleteBook } from '../redux/books/booksSlice';
+import '../styles/BookContainer.css';
 
 const BookContainer = ({
   item, title, author, category,
 }) => {
-  const [complete, setComplete] = useState('0');
   const dispatch = useDispatch();
 
-  useEffect(() => {
-    setComplete('90%');
-  }, []);
-
   const handleRemoveBook = (id) => {
     dispatch(deleteBook(id));
   };
 
+  const percentage = 80;
+
   return (
     <div className="book-container">
       <div className="book-details">
-        <div className="book-category">{category}</div>
-        <div className="book-title">{title}</div>
-        <div className="book-author">{author}</div>
+        <div className="category">{ category }</div>
+        <div className="book-title">{ title }</div>
+        <div className="book-author">{ author }</div>
         <div className="book-buttons">
-          <button type="button" className="book-btn">
-            Comments
-          </button>
+          <button type="button" className="book-btn">Comments</button>
           <button
```

Comment:
- [ ] it would be better to change the background color of the page books to a bit gray as in the Design in order to match it 
- Yours 

<img width="1436" alt="Screenshot 2023-05-19 at 16 41 12" src="https://github.com/Bellagirl-maker/book-store/assets/61790587/edf837e9-86ed-4caa-ac95-9e47a8ea03d3">

- Design 

<img width="950" alt="Screenshot 2023-05-19 at 16 36 20" src="https://github.com/Bellagirl-maker/book-store/assets/61790587/d7deb4b9-016b-4a67-93a5-59345dd1dec4">

Reply:
@youmari done withi this change